cmd/alerts/list_alerts.go

```diff
@@ -0,0 +1,138 @@
+package alerts
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	levelCmd  string
+	cmdStatus []string
+)
+
+type alertCmd struct {
+	clusterID  string
+	alertLevel string
+	active     bool
+}
+
+type alertJson struct {
+	Labels struct {
+		Alertname string `json:"alertname"`
+		Severity  string `json:"severity"`
+	} `json:"labels"`
+
+	Status struct {
+		State string `json:"state"`
+	} `json:"status"`
+
+	Annotations struct {
+		Summary string `json:"summary"`
+	} `json:"annotations"`
```

Comment:
I would suggest moving these into separate structs outside the main alertJSON struct and then refer them. Makes it easier to read